java-squid/src/main/java/org/sonar/java/ast/parser/ListTreeImpl.java

```diff
@@ -61,10 +72,35 @@ public Kind getKind() {
 
   @Override
   public Iterator<Tree> childrenIterator() {
-    // TODO
-    throw new UnsupportedOperationException();
+    //FIXME SONARJAVA-547 Separators
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)